src/builders/go/gomod2nix/default.nix

```diff
@@ -0,0 +1,21 @@
+{
+  lib,
+  pkgs,
+  externals,
+  ...
+}:
+
+{
+  fetchedSources,
+  dreamLock,
+}:
+let
+  gomod2nixTOML = fetchedSources.mapAttrs
+    lib.replaceStrings ["__slash__"] ["/"] dependencyObject.goName;
```

Comment:
Actually I think we should keep the original names here. It is better if the dream-lock contains the original names, because this will allow us to write override using the upstream names. This should prevent confusion for maintainers.
The builder can later use `utils.sanitizeDerivationName` when creating the derivations.

Reply:
Done!